frappe/patches/v16_0/webhook_docevent_unscrub.py

```diff
@@ -0,0 +1,9 @@
+import frappe
+
+
+def execute():
+	webhooks = frappe.get_all("Webhook", ["name", "webhook_docevent"])
+	for webhook in webhooks:
+		frappe.db.set_value(
+			"Webhook", webhook.name, "webhook_docevent", frappe.unscrub(webhook.webhook_docevent)
```

Comment:
Don't do such unnecessary breaking changes.

Reply:
https://gameplan.frappe.cloud/g/space/94/discussion/2197/policy-on-breaking-changes-and-branching

Reply:
More context:
- chaniging business logic related select field breaks any code that was operating on this. People have dynamic webhooks (wild, I know)
- This breaks importing fixtures that were defined before. 

In general, only break something if the cost of not breaking it far outweigh the cost of dealing with breakages.

Reply:
Costs don't justify the benefits (consistency?), just revert the changes IMO.